sparklightautoml/computations/builder.py

```diff
@@ -0,0 +1,105 @@
+from typing import Optional, Tuple, Dict, Any, Union
+
+from sparklightautoml.computations.base import ComputationsManager, ComputationsSettings
+from sparklightautoml.computations.sequential import SequentialComputationsManager
+from sparklightautoml.computations.parallel import ParallelComputationsManager
+
+# either named profile and parallelism or parallelism settings or factory
+AutoMLComputationsSettings = Union[Tuple[str, int], Dict[str, Any], 'ComputationManagerFactory']
+
+
+class ComputationsManagerFactory:
+    def __init__(self, computations_settings: Optional[Union[Tuple[str, int], Dict[str, Any]]] = None):
+        super(ComputationsManagerFactory, self).__init__()
+        computations_settings = computations_settings or ("no_parallelism", -1)
+
+        if isinstance(computations_settings, Tuple):
+            mode, parallelism = computations_settings
+            self._computations_settings = build_named_parallelism_settings(mode, parallelism)
+        else:
+            self._computations_settings = computations_settings
+
```

Comment:
lets rewrite code below
`
settings = self._computations_settings
self._some_param = settings.get('some_param')
`
settings.get('some_param', None) equivalent to settings.get('some_param')

Reply:
fixed